common/src/main/java/com/wynntils/features/user/overlays/ObjectivesOverlayFeature.java

```diff
@@ -102,19 +102,26 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
                         case Bottom -> this.getHeight() - renderedHeight;
                     };
 
+            final String text =
+                    guildObjective.getGoal() + ": " + guildObjective.getScore() + "/" + guildObjective.getMaxScore();
             FontRenderer.getInstance()
                     .renderAlignedTextInBox(
                             poseStack,
-                            guildObjective.getGoal() + ": " + guildObjective.getScore() + "/"
-                                    + guildObjective.getMaxScore(),
+                            text,
                             this.getRenderX(),
                             this.getRenderX() + this.getWidth(),
                             renderY,
-                            0,
+                            this.getWidth(),
                             this.textColor,
                             FontRenderer.TextAlignment.fromHorizontalAlignment(this.getRenderHorizontalAlignment()),
                             this.textShadow);
 
+            float height = FontRenderer.getInstance().calculateRenderHeight(List.of(text), this.getWidth());
+
+            if (height > 9) {
+                renderY += height - 9;
```

Comment:
We don't need offsetY here? (Not sure what it does, just noticed the lack of symmetry)

Reply:
There is only a singular guild objective, offsetX is used to render multiple daily objectives.